ddp-workspace/projects/ddp-rarex/src/app/components/top-bar/top-bar.component.ts

```diff
@@ -65,4 +94,14 @@ export class TopBarComponent implements OnInit {
         this.profile = profile;
       });
   }
+
+  private initRouteListener(): void {
+    const routeSub = this.router.events
+      .pipe(filter(e => e instanceof NavigationEnd))
+      .subscribe((e: NavigationEnd) => {
+        this.isParticipantNameShown = e.url === `/${RoutePaths.Survey}`;
```

Comment:
Does this mean we're only showing name on when viewing the surveys? What about the dashboard?